src/redux/actionCreators/resetAction.js

```diff
@@ -0,0 +1,58 @@
+import axios from 'axios';
+import i18next from 'i18next';
+import { toast } from 'react-toastify';
+import {
+  FORGOT_FAILED,
+  FORGOT_PENDING,
+  FORGOT_SUCCESS,
+  RESET_FAILED,
+  RESET_PENDING,
+  RESET_SUCCESS,
+} from '../actionTypes/actionTypes';
+
+const server = 'https://incredible-phantom.herokuapp.com';
```

Comment:
you can initialize this backend URL as a constant in the helper folder so that it can be reusable instead of everyone initializing it in their own files

Reply:
or put it in .env

Reply:
Or let me do it the way suggested